src/overlays/actors/ovl_Obj_Rotlift/z_obj_rotlift.c

```diff
@@ -28,24 +30,108 @@ const ActorInit Obj_Rotlift_InitVars = {
     (ActorFunc)ObjRotlift_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96178[] = {
+struct AnimatedThing D_80B96160[] = {
+    {
+        object_rotlift_DL_000400,
+        object_rotlift_Matanimheader_001F98,
+        &object_rotlift_Colheader_002190,
+    },
+    {
+        object_rotlift_DL_002CE0,
+        object_rotlift_Matanimheader_004A08,
+        &object_rotlift_Colheader_004DF0,
+    },
+};
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 800, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 800, ICHAIN_STOP),
 };
 
-#endif
+void func_80B95E20(ObjRotlift* this) {
+    Actor* actor;
+    Actor** actorUpdate;
+    f32 rotShift;
+    s32 i;
+    actorUpdate = this->actorPtr;
+
+    for (i = 0, rotShift = 300.0f; i < 2; i++, actorUpdate++) {
+        actor = *actorUpdate;
+        if (actor->update == NULL) {
+            *actorUpdate = NULL;
+        } else {
+            actor->world.pos.x = (Math_SinS(this->dyna.actor.shape.rot.y) * rotShift) + this->dyna.actor.world.pos.x;
+            actor->world.pos.y = this->dyna.actor.world.pos.y + (380.0f * this->dyna.actor.scale.y);
+            actor->world.pos.z = (Math_CosS(this->dyna.actor.shape.rot.y) * rotShift) + this->dyna.actor.world.pos.z;
+            actor->shape.rot.y = this->dyna.actor.shape.rot.y;
+        }
+        rotShift -= 600.0f;
+    }
+}
+
+void ObjRotlift_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    ObjRotlift* this = THIS;
+    GlobalContext* globalCtx = globalCtx2;
+    s32 params;
+    s32 spawnParams;
+    s32 i;
+    AnimatedThing* animated;
+    Actor** actor;
+
+    params = OBJROTLIFT_GET_1(thisx);
+    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
+    if (params == 0) {
+        for (actor = this->actorPtr, i = 0; i < 2; i++, actor++) {
+            if (!(OBJROTLIFT_GET_4000(thisx)) || (i)) {
```

Comment:
```suggestion
            if (!OBJROTLIFT_GET_4000(thisx) || (i != 0)) {
```
The parens around `OBJROTLIFT_GET_4000(thisx)` are unnecessary. Also since `i` is not a boolean, the explict check against 0 should stay.